tox.ini

```diff
@@ -23,9 +25,10 @@ setenv =
     !image: MPLFLAGS =
     clocale: LC_CTYPE = C.ascii
     clocale: LC_ALL = C
-    devdeps: PIP_VERBOSE = 1
-    devdeps: PIP_EXTRA_INDEX_URL = https://pypi.anaconda.org/liberfa/simple https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
-    mpldev: PIP_EXTRA_INDEX_URL = https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
+    devdeps: UV_INDEX = https://pypi.anaconda.org/liberfa/simple https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
+    devdeps: UV_INDEX_STRATEGY = unsafe-best-match # match pip's behavior
+    mpldev: UV_INDEX = https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
+    mpldev: UV_INDEX_STRATEGY = unsafe-best-match # match pip's behavior
```

Comment:
```suggestion
    devdeps, mpldev: UV_INDEX_STRATEGY = unsafe-best-match # match pip's behavior
    mpldev: UV_INDEX = https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
```

Reply:
thanks ! done in a slightly different order so that both `UV_INDEX` lines sit next to each other.